### ISSUE TITLE
Ability to use Realm Studio as non admin user

### ISSUE THREAD

**Issue by pfedotovsky** [realm/realm-studio#1143]
**Describe the solution you'd like**
I'd like to use Realm Studio without being admin user.

I'd like to share my realms with other but I don't want to give admin permissions for everyone. It would be nice if users can login to Realm Studio and manage realms without admin permissions

**Comment by kraenhansen** [comment]
What features would you expect from connecting to a server with a non-admin users? Primarily listing and opening the Realms the users has access to?

**Comment by pfedotovsky** [comment]
Yes, I expect that non-admin user can view and edit realms that they have access to
